dist/react-json-schema.js

```diff
@@ -34,7 +34,7 @@ var ReactJsonSchema = function () {
   }, {
     key: 'parseSubSchemas',
     value: function parseSubSchemas() {
-      var subSchemas = arguments.length <= 0 || arguments[0] === undefined ? [] : arguments[0];
+      var subSchemas = arguments.length > 0 && arguments[0] !== undefined ? arguments[0] : [];
```

Comment:
Seems like this is compiling different? I assume this is ok and its just the result of the compiler version...